src/Elcodi/Component/Rule/Repository/RuleRepository.php

```diff
@@ -27,14 +27,14 @@
 class RuleRepository extends EntityRepository
 {
     /**
-     * Return a rule by name
+     * Return a rule by id
      *
-     * @param string $name
+     * @param string $id
      *
      * @return RuleInterface|null
      */
-    public function findOneByName($name)
+    public function findOneById($id)
     {
-        return parent::findOneBy(['name' => $name]);
+        return parent::findOneBy(['id' => $id]);
```

Comment:
I don't really understand this method. Without this implementation you can call to `findOneById` as well, and the result will be the same. In fact, you can call `find` and will return the same result...